Gemfile

```diff
@@ -33,7 +33,7 @@ gem 'notifications-ruby-client'
 if ENV['API_DEV']
   gem 'gds-api-adapters', path: '../gds-api-adapters'
 else
-  gem 'gds-api-adapters', '47.2'
+  gem 'gds-api-adapters', '~> 47.2.1'
```

Comment:
You can just ease this down to `~> 47.2` which will allow updating up to (but excluding) version 48 which fits semvar approach of gds-api-adapters.

Reply:
Done 👍